### ISSUE TITLE
Can your deploy a version on huggingface space ?

### ISSUE THREAD

**Issue by svjack** [zideliu/StyleDrop-PyTorch#1]
Can you deploy your gradio_demo on on huggingface space ?

**Comment by loboere** [comment]
x2

**Comment by chenxwh** [comment]
did you try with `python gradio_demo.py`? I am getting very random output

**Comment by zideliu** [comment]
> did you try with `python gradio_demo.py`? I am getting very random output

U R right, I forgot to set the seed at the beginning. I have fixed this problem in the latest code [commit 20eff45](https://github.com/zideliu/StyleDrop-PyTorch/commit/20eff45767900ba424ce6589cbb66a9b192e9c9)

**Comment by zideliu** [comment]
> Can you deploy your gradio_demo on on huggingface space ?

On the way

**Comment by chenxwh** [comment]
> > did you try with `python gradio_demo.py`? I am getting very random output
> 
> U R right, I forgot to set the seed at the beginning. I have fixed this problem in the latest code [commit 20eff45](https://github.com/zideliu/StyleDrop-PyTorch/commit/20eff45767900ba424ce6589cbb66a9b192e9c9)


Hm, I still see the same issue

<img width="726" alt="Screenshot 2023-07-05 at 09 20 00" src="https://github.com/zideliu/StyleDrop-PyTorch/assets/70536672/6bc58895-8c5d-4fbe-9882-ccd10fd6a30b">

**Comment by zideliu** [comment]
> > > did you try with `python gradio_demo.py`? I am getting very random output
> 
> > 
> 
> > U R right, I forgot to set the seed at the beginning. I have fixed this problem in the latest code [commit 20eff45](https://github.com/zideliu/StyleDrop-PyTorch/commit/20eff45767900ba424ce6589cbb66a9b192e9c9)
> 
> 
> 
> 
> 
> Hm, I still see the same issue, does it look like I have set up something else wrong?
> 
> 
> 
> <img width="726" alt="Screenshot 2023-07-05 at 09 20 00" src="https://github.com/zideliu/StyleDrop-PyTorch/assets/70536672/6bc58895-8c5d-4fbe-9882-ccd10fd6a30b">
> 
> 
> 
> 

If it's convenient, can you put out your log？ Make sure you have downloaded the cc3m pre-trained model and put it in `assets/ckpts/`

**Comment by chenxwh** [comment]
ah thanks, a typo indeed, saved in `ckpt` instead

**Comment by encounter1997** [comment]
The gradio demo is now deployed to HuggingFace space, available [here](https://huggingface.co/spaces/zideliu/styledrop).

**Comment by zideliu** [comment]
Fixed
